cassandra-mesos-scheduler/src/main/java/io/mesosphere/mesos/frameworks/cassandra/scheduler/CassandraScheduler.java

```diff
@@ -88,6 +93,9 @@ public void resourceOffers(final SchedulerDriver driver, final List<Offer> offer
         if (LOGGER.isDebugEnabled()) {
             LOGGER.debug("> resourceOffers(driver : {}, offers : {})", driver, protoToString(offers));
         }
+
+        configuration.getSeedManager().syncSeeds(clock.now().getMillis());
```

Comment:
I don't think this should be called on every resource offer.  Instead `SeedManager` should be updated to have it's constructor receive a `java.util.concurrent.ScheduledExecutorService` and to start a background task to call `syncSeeds` on a periodic basis.

And example elsewhere in the code that is following this same pattern can be found in [`CassandraExecutor#startCheckingHealth`](https://github.com/mesosphere/cassandra-mesos/blob/master/cassandra-mesos-executor/src/main/java/io/mesosphere/mesos/frameworks/cassandra/executor/CassandraExecutor.java#L514-L519)